contentdm_exporter/contentdm_record_exporter.py

```
@@ -335,7 +376,7 @@ def get_file_metadata_json(file_level_id, collection_alias):
     return file_metadata
 
 
-def get_bib_record(collection_alias, cdm_recid):
+def get_bib_record(collection_alias, cdm_recid, field_mapping={}):
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)